### ISSUE TITLE
Minecraft Crash When Applying Video Settings

### ISSUE THREAD

**Issue by TheButterbrotMan** [MCTeamPotato/RuOK#9]
The game crashes with a NoClassDefFoundError in the RuOK mod when attempting to apply changes in the Embeddium video options screen. This issue arises specifically when clicking "Done" after modifying settings, leading to an error in parsing the double value "73,8," which uses a comma as a decimal separator instead of a period. 

For reference, the crash occurs in Minecraft version 1.20.1 with Forge version 47.3.0 and RuOK version 1.6.1 while using Embeddium version 0.3.31.

**Comment by Yuqi154** [comment]
Hello,


Thank you for reporting the issue. To help diagnose and resolve the problem, could you please provide the latest.log file from your Minecraft installation? This will give us more insight into what's causing the NoClassDefFoundError and the issue with parsing the double value.


Thank you for your cooperation!

**Comment by TheButterbrotMan** [comment]
I apologize for forgetting to attach the crash log. Here it is:

If you still need it, I can add the latest.log later.

You can find the log at the following link: [Crash Log](https://gist.github.com/TheButterbrotMan/d3faed8c494cad71e4731761b0d804ed).

**Comment by AweiMC** [comment]
Hello,

Thank you for your patience while we investigated the issue. Our analysis has indicated that a conflict with the "Not Enough Crashes" mod may be causing your game to crash incorrectly and misidentify the source of the problem as your mod (RuOK).

To help resolve this issue, we recommend trying to remove the "Not Enough Crashes" mod from your Minecraft installation and see if the crash persists. This step may help isolate the cause of the problem.

Please let us know if removing the mod resolves the issue or if you encounter any further problems. Thank you for your understanding and cooperation!

**Comment by AweiMC** [comment]
Thank you for providing the log!

After analyzing the log you provided, I have determined a possible cause: NEC (Not Enough Crashes) may be incorrectly intercepting the RuOK mod.

I recommend trying to remove the NEC (Not Enough Crashes) mod to resolve this issue.

Thank you!

**Comment by TheButterbrotMan** [comment]
Yes, I have removed the mod, but it still crashes. Here's the log:

https://gist.github.com/TheButterbrotMan/549ff2ab5d7d5a56bf95db3b56a9f0e4
